pkg/checksum/checker.go

```diff
@@ -292,10 +292,10 @@ func (c *Checker) replaceChunk(chunk *table.Chunk) error {
 	c.recopyLock.Lock()
 	defer c.recopyLock.Unlock()
 
-	if _, err := dbconn.RetryableTransaction(context.TODO(), c.db, false, dbconn.NewDBConfig(), deleteStmt); err != nil {
+	if _, err := dbconn.RetryableTransaction(ctx, c.db, false, dbconn.NewDBConfig(), deleteStmt); err != nil {
```

Comment:
Hope there's no unintended side effects here. I don't think this is a problem but want to highlight this change. It's to satisfy the `contextcheck` linter.

Reply:
It looks good to me. I think this is a code improvement.